services/email/index.js

```diff
@@ -168,10 +168,11 @@ Emailer.prototype.send = function send(email, callback) {
       }
     }
 
-    logger.info('Emailing caseworker: ', email.subject);
+    logger.info(config.email.duplicate ? 'Emailing Caseworker Duplicate inbox: ' +
```

Comment:
Has we already removed the config.email.duplicate reference out of config.js on line 25?